### PR TITLE
[TRANSLATION] Spanish full revision of description and some names

### PR DESCRIPTION
**Pull request by julenuri**
Full translation revision.

I'll be glad if you merge it.

### CODE REVIEW

File: watercolor.txt [hunk 1]
```diff
@@ -37,3 +37,8 @@ Size = 861,1 KiB
 License = 免费软件
 Description = 基于 Windows 代号 Whistler 外观的 Windows 主题。
 
+[Section.0a]
+Description = Un tema de Windows basado en el look del Windows Codename Whistler.
+
+
+
```
Comment (learn-more) [inline watercolor.txt:44]:
```suggestion
```

File: iqpuzzle.txt [hunk 2]
```diff
@@ -11,6 +11,8 @@ SHA1 = 300dcdb2acb59175110f196c6f27b5441d54eb42
 CDPath = none
 SizeBytes = 9998445
 
+[Section.0a]
+Description = Un puzzle pentominó que te divertirá y retará intelectualmente.
 [Section.0407]
```
Comment (learn-more) [inline iqpuzzle.txt:16]:
```suggestion
Description = Un puzzle pentominó que te divertirá y retará intelectualmente.

[Section.0407]
```

Comment (julenuri) [reply iqpuzzle.txt]: Done

File: watercolor.txt (hunk 1 above)
Comment (julenuri) [reply watercolor.txt]: Done.

**Pull request merged**